### PR TITLE
feat(ui): add node memoization information to node summary view

### PR DESCRIPTION
**Pull request by rbreeze**
Screen Shot 2020-08-11 at 4 46 32 PM

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [x] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [x] I've signed the CLA.
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My builds are green. Try syncing with master if they are not. 
* [x] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

### CODE REVIEW

File: ui/src/app/shared/components/memoization-info/memoization-info.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@import 'node_modules/argo-ui/src/styles/config';
+
+.mi {
```
Comment (alexec) [inline ui/src/app/shared/components/memoization-info/memoization-info.scss:3]: anyway we can avoid more styles please?

File: ui/src/app/shared/components/memoization-info/memoization-info.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  padding: 1em 0;
+  &--row {
+    display: flex;
+    line-height: 2em;
+    &__label, &__data {
+      padding: 0 10px;
+    }
+    &__label {
+      font-weight: 600;
+      border-right: 2px solid $argo-color-gray-3;
```
Comment (simster7) [inline ui/src/app/shared/components/memoization-info/memoization-info.scss:13]: I don't think the bold looks good given that nothing else in this screen is bolded

File: ui/src/app/shared/components/memoization-info/memoization-info.scss (hunk 1 above)
Comment (rbreeze) [reply ui/src/app/shared/components/memoization-info/memoization-info.scss:3]: Yeah will do!

File: ui/src/app/shared/components/memoization-info/memoization-info.scss (hunk 2 above)
Comment (rbreeze) [reply ui/src/app/shared/components/memoization-info/memoization-info.scss:13]: Good point. Going to remove styles as @alexec suggested, will address in the process

**Comment by rbreeze** [comment]
Screen Shot 2020-08-12 at 10 26 37 AM

**Comment by alexec** [comment]
I think the original layout looked better :(

What about the CSS we use for resource duration in the drawer?

**Comment by rbreeze** [comment]
@alexec I'll explore

**Comment by rbreeze** [comment]
Factored out relevant portion into `InlineTable` component. Goal with Resource Duration in Workflows Drawer was to remain the same visually, as seen below: 

Screen Shot 2020-08-14 at 5 39 00 PM

and the Memoization info is visually similar to the first iteration but without bolding, as per Simon's suggestion. 

Screen Shot 2020-08-14 at 5 38 48 PM

**Pull request merged**